ml-agents/mlagents/trainers/bc/offline_trainer.py

```diff
@@ -15,7 +15,8 @@
 class OfflineBCTrainer(BCTrainer):
```

Comment:
Does the new CSV thing works with BC as well or just PPO ?

Reply:
Just PPO